### PR TITLE
feat: update banner text

### PR DESCRIPTION
**Pull request by paulatulis**
Updated text in top banner to push people to demo page. Feel free to suggest alternate wording!

Screen Shot 2020-02-07 at 12 36 42 PM

### CODE REVIEW

**Comment by paulatulis** [comment]
[Banner: update language to be demo related rather than about Studio](https://trello.com/c/418FUWGF/357-banner-update-language-to-be-demo-related-rather-than-about-studio)

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 857b50867083f50a02a3ce4ef460cbcec8bc2e61

https://deploy-preview-1350--stoplightio.netlify.com

**Comment by falsaffa** [comment]
Maybe "Ready to accelerate **your team's** API development by 36%?". And the 36% where is that number from?

File: netlify/settings.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 banners:
   - ends: 1609430400000
     markdown: >-
-      Announcing Stoplight Studio, our most powerful API Designer yet, now with OpenAPI v3 support! <u class='ml-1'>**[Learn More](https://stoplight.io/studio)**</u>
+      Ready to accelerate API Development by 36%? <u class='ml-1'>**[Request a Demo](https://stoplight.io/demo)**</u>
```
Comment (paulatulis) [inline netlify/settings.yaml]:
```suggestion
      Ready to accelerate your team's API Development by 36%? <u class='ml-1'>**[Request a Demo](https://stoplight.io/demo)**</u>
```

**Comment by paulatulis** [comment]
@falsaffa made the change. As far as where the number comes from, I am unsure. It's been on our demo page since before I started, I believe. If we don't have any stats we can link to an actual source, I can change it to be less specific.

**Pull request merged**